Create [WIP] RNA-seq - New Tools, 2016 Q3

vkcshl opened the pull request:
This will be updated w.r.t scope of use-cases and dependencies, after additional feedback from the Euks team.

vkcshl committed:
Create [WIP] RNA-seq - New Tools, 2016 Q3

This will be updated w.r.t scope of use-cases and dependencies, after additional feedback from the Euks team.

r2sunita commented:
Vivek, 

Here are the use cases for new tools. Please merge these. 

1. As a user, I would like to get the greatest number of correctly assigned reads with great speed and with a low memory footprint, I would like to use HiSAT2 to conduct sample read alignments against a reference genome.

2. As a user, I would like to do alignment of reads containing SNPs, I would like to use HiSAT2 for this analysis. 

3. As a user, I would like to use fast and highly efficient assembler that can use the alignments of raw reads from existing alignment tool TopHat or new alignment tool – HiSAT2.  Based on the literature, on a simulated dataset, StringTie resulted 20% more transcriptome assembly than Cufflinks and it runs faster on all data sets compared with other assembly software.

4. As a user, I would like to use Ballgown tool to do estimation of differential expression of genes or transcripts from RNA-seq experiments. I would like to use Ballgown because it provides fast, flexible differential expression analysis and various options to explore the transcriptome assembly. It also provides functionality for visualization of differentially expressed results. Ballgown is also compatible with TopHat/Cufflinks results.

scanon commented:
For a PD I think we need to be more specific than candidates.  We should narrow in on what is in scope.

vkcshl committed:
Update [WIP] RNA-seq - New Tools, 2016 Q3

Since we are increasingly using PDs as planning documents in addition to product specs, I have updated it to only the tools we plan to complete in two sprints allocated to this effort and if time permits add new ones. I have still kept the visualization tools (with a note that it is not guaranteed to be delivered in Q3).

FastQC and other read mgmt. tools, would be covered in another PD.

scanon commented:
This looks better.  We should add a mock narrative and any mockups for the Viz.
